codespell_lib/data/dictionary.txt

```diff
@@ -1255,6 +1257,7 @@ campagin->campaign
 campain->campaign
 campaing->campaign
 campains->campaigns
+cancelation->cancellation
```

Comment:
`cancelation` and `cancellation` are both correct https://github.com/lucasdemarchi/codespell/issues/103#issuecomment-341706499